xbmc/windowing/windows/WinSystemWin32.cpp

```diff
@@ -80,8 +80,11 @@ CWinSystemWin32::CWinSystemWin32()
   CAESinkDirectSound::Register();
   CAESinkWASAPI::Register();
   CWin32PowerSyscall::Register();
-  m_irss.reset(new CIRServerSuite());
-  m_irss->Initialize();
+  if (CServiceBroker::GetSettings().GetBool("input.irserversuite"))
```

Comment:
For new settings we usually introduce a string constant fo the name of the new stetting in https://github.com/xbmc/xbmc/blob/master/xbmc/settings/Settings.h / https://github.com/xbmc/xbmc/blob/master/xbmc/settings/Settings.cpp

Just saying... ;-)